assessments/serializer.py

```
@@ -1,85 +1,166 @@
+        fields = "__all__"
 
-class ExamQuestionsSerializer(BaseSerializer):
-    class Meta(BaseSerializer.Meta):
-        model = models.ExamQuestions
-        fields = '__all__'
 
 class QuestionOptionsSerializer(BaseSerializer):
     class Meta(BaseSerializer.Meta):
         model = models.QuestionOptions
-        fields = '__all__'
+        fields = ["id", "question", "option_code", "option_text", "is_correct"]
```

Ruff (0.15.18)

[warning] Mutable default value for class attribute

(RUF012)